### PR TITLE
Add one space before </strong> tag.

### LINKED ISSUE #3409: fix: add space after colon in deadline field of assignment
**Describe the bug**
In the show page of assignment, all the details are spaced after the colon(:) except the Deadline field.

**To Reproduce**
1. Go to 'show page of assignment'
2. See error

**Expected behavior**
Add space after colon in deadline field of assignment details.

**Steps to solve**
1. Go to the show [page](https://github.com/CircuitVerse/CircuitVerse/blob/a92acd5f460092ee4a803b857745e4c2793b62c3/app/views/assignments/show.html.erb#L24)
2. Add one space before `</strong>` tag.

**Screenshots**
Screenshot 2022-12-17 at 6 34 22 PM

**Additional context**
Great for first time contributors to Circuitverse

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by Malavi1**
Fixes #3409 

#### Describe the changes you have made in this PR -
space added in front of the strong tag

### Screenshots of the changes (If any) -
space added

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by VaibhavUpreti** [comment]
@Malavi1 instead of giving a literal space, the hound warning can be solved by writing `&nbsp;` instead of empty space.

**Comment by Malavi1** [comment]
Thankyou @VaibhavUpreti for letting me know I will change

File: app/views/assignments/show.html.erb [hunk 1]
```diff
@@ -21,7 +21,7 @@
         <div class="assignments-data">
           <p><%= t("assignments.assignment_data.name_html", name: @assignment.name) %></p>
           <p><%= t("assignments.assignment_data.group_html", group_name: @assignment.group.name) %></p>
-          <p><strong><%= t("assignments.deadline") %></strong><span class="assignments-deadline" data-year=<%= deadline_year(@assignment) %>
+          <p><strong><%= t("assignments.deadline") %>&nbsp</strong><span class="assignments-deadline" data-year=<%= deadline_year(@assignment) %>
```
Comment (VaibhavUpreti) [inline app/views/assignments/show.html.erb:24]:
```suggestion
          <p><strong><%= t("assignments.deadline") %>&nbsp;</strong><span class="assignments-deadline" data-year=<%= deadline_year(@assignment) %>
```

**Comment by Malavi1** [comment]
@VaibhavUpreti one was failing to pass, but actually there is nothing at line 10

**Review by VaibhavUpreti — APPROVED** [review]

**Pull request closed without merging**